{"name":"C. Under Attack!!!","group":"Codeforces - A2SV G5 - Contest #4","url":"https://codeforces.com/gym/494181/problem/C","interactive":false,"memoryLimit":256,"timeLimit":2000,"tests":[{"id":1703314688744,"input":"4\n4 4\n1 2 2 1\n2 4 2 4\n2 2 3 1\n2 4 2 4\n2 1\n1\n0\n3 3\n1 1 1\n1 1 1\n1 1 1\n3 3\n0 1 1\n1 0 1\n1 1 0","output":"20\n1\n5\n3"}],"testType":"single","input":{"type":"stdin"},"output":{"type":"stdout"},"languages":{"java":{"mainClass":"Main","taskClass":"CUnderAttack"}},"batch":{"id":"e501b9c4-53d1-403b-8142-f68cdf87e1c4","size":1},"srcPath":"/home/amanigar/Documents/AmaniFarms/cp/C_Under_Attack.py"}